pkg/ccl/changefeedccl/helpers_test.go

```diff
@@ -62,8 +63,8 @@ func waitForSchemaChange(
 	})
 }
 
-func readNextMessages(f cdctest.TestFeed, numMessages int, stripTs bool) ([]string, error) {
-	var actual []string
+func readNextMessages(f cdctest.TestFeed, numMessages int) ([]cdctest.TestFeedMessage, error) {
+	var actual []cdctest.TestFeedMessage
```

Comment:
return as `TestFeedMessage` here because we need messages in struct format to verify ordering guarantees